mlir/utils/jenkins/Jenkinsfile.nightly-perf

```diff
@@ -20,19 +20,21 @@ pipeline {
         stage('Build MIOpenDriver') {
             steps {
                 dir('MIOpen') {
+                    sh 'mkdir install'
                     git branch: 'develop', poll: false,\
                                 url: 'https://github.com/ROCmSoftwarePlatform/MIOpen.git'
-                    cmake arguments: "-P install_deps.cmake --minimum",\
+                    cmake arguments: "-P install_deps.cmake --minimum --prefix ./MIOpen_deps",\
                                 installation: "InSearchPath"
                     cmakeBuild buildDir: 'build',\
                                 buildType: 'Release',\
                                 installation: 'InSearchPath',\
                                 cmakeArgs: '''
                                  -DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++
                                  -DMIOPEN_BACKEND=HIP
-                                 -DCMAKE_PREFIX_PATH="/opt/rocm/hip;/opt/rocm/;/usr/local" ..
+                                 -DCMAKE_PREFIX_PATH="${WORKSPACE}/MIOpen/MIOpen_deps"
+                                 -DCMAKE_INSTALL_PREFIX=../install/ ..
                                 '''
-                    sh 'cd build; make -j $(nproc) MIOpenDriver'
+                    sh 'cd build; make -j install'
```

Comment:
Could you still have a `make -j MIOpenDriver` followed by a `make install` to isolate the steps (and since some `install` Makefiles don't build before installs). I'd also argue we don't want to `-j` on install, just to be safe.

Reply:
I'd want the build and install steps being split up for clarity

Reply:
Sounds reasonable.